tests/scripts/kruize_metrics.py

```diff
@@ -0,0 +1,357 @@
+            if "result" in results_data[key] and isinstance(results_data[key]["result"], list) and len(results_data[key]["result"]) > 0:
+                if "value" in results_data[key]["result"][0]:
+                    results_map[key] = results_data[key]["result"][0]["value"][1]
+            # Uncomment else part to debug which query is not working.
+            #else:
+            #    print(f"Failed to run query '{query}' with status code {response.status_code}")
+    kruize_results, db_size = get_postgresql_metrics(namespace)
+    results_map["kruize_results"] = kruize_results
+    results_map["db_size"] = db_size
+    return results_map
+
+
+def write_header_to_csv(filename):
+    if not os.path.exists(filename) or os.path.getsize(filename) == 0:
+        with open(filename, 'w') as f:
+            writer = csv.DictWriter(f, fieldnames=csv_headers)
+            writer.writeheader()
+
+def job(queries_type,outputdir):
+    now_utc = datetime.utcnow()
+    timestamp_utc = now_utc.isoformat()
+    if queries_type == "increase":
+        outputfile = os.path.join(outputdir, "increase_" + resultsfile)
+        print("RUNNING THE JOB TO COLLECT KRUIZE INCREASE METRICS..")
+        results_map = run_queries("increase")
+    elif queries_type == "total":
+        outputfile = os.path.join(outputdir, "total_" + resultsfile)
+        print("RUNNING THE JOB TO COLLECT KRUIZE TOTAL METRICS..")
+        results_map = run_queries("total")
+    results_map['timestamp'] = timestamp_utc
+    write_header_to_csv(outputfile)
+    with open(outputfile, 'a') as f:
+        writer = csv.DictWriter(f, fieldnames=csv_headers)
+        writer.writerow(results_map)
+
+def schedule_job(queries_type):
+    outputdir = "results"
+    if not os.path.exists(outputdir):
+        os.mkdir(outputdir)
+    numeric_time = int(time_duration[:-1])
+    time_in_seconds = numeric_time * 60
+    if getOneDataPoint == "true":
+        job("increase",outputdir)
+        job("total",outputdir)
+    else:
+        if duration is None:
+            while True:
+                job(queries_type,outputdir)
+                print("Sleep for ",time_in_seconds, " seconds")
+                time.sleep(time_in_seconds)
+        else:
+            now = datetime.utcnow()
+            end = now + timedelta(hours=duration)
+            while now < end:
+                now = datetime.utcnow()
+                job(queries_type,outputdir) 
+                print("Sleep for ",time_in_seconds, " seconds")
+                time.sleep(time_in_seconds) 
+
+def main(argv):
+    global duration
+    global cluster_type
+    global server
+    global clusterResults
+    global time_duration 
+    global queries_map
+    global getOneDataPoint
+    global resultsfile
+    global namespace
+
+    try:
+        opts, args = getopt.getopt(argv,"h:c:s:d:t:q:p:r:")
+    except getopt.GetoptError:
+        print("kruize_metrics.py -c <cluster type> -s <server>")
```

Comment:
If you dont provide any parameters, the script should print the help message

Reply:
It is not clear what is the server here for a first time user, please elaborate in the help message

Reply:
Since cluster type is fixed, print the options

Reply:
If it is unable to access the server port, it should catch the error and print a helpful message as this would be the most common error

Reply:
Done.